pkg/service/rtcservice.go

```diff
@@ -229,15 +229,22 @@ func (s *RTCService) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// give it a few attempts to start session
 	var cr connectionResult
 	var initialResponse *livekit.SignalResponse
-	for i := 0; i < 3; i++ {
+	for i := 0; r.Context().Err() == nil; i++ {
```

Comment:
Does this mean this keeps going till the client cancels?

Reply:
that's correct. though based on dz's latest review, I'll revert this to only trying 3 times